gke/README.md

```diff
@@ -0,0 +1,187 @@
+# Bigtable auto-scaler Deployment
+
+## General
+
+In this guide we will deploy Bigtable auto-scaler into GKE.
+
+![Bigtable autoscaler on GKE](./img/bigtable-autoscaler.svg)
+
+## Getting started
+
+### Prerequisites
+
+* Bigtable cluster (or several) to autoscale
+* GKE cluster
+* Service account JSON key that has relevant access to the Bigtable clusters to autoscale.
+  * If the autoscaler is running in the same GCP project as all the Bigtable clusters, the Compute Engine Default Service Account is sufficient.
+  * The minimum permissions are:
+    * Role Bigtable Administrator, in particular the permissions
+      * bigtable.clusters.get
+      * bigtable.clusters.update
+    * Role Monitoring Viewer, in particular the permissions
+      * monitoring.timeSeries.list
+* Docker
+* Java 11 and maven
+* PostgreSQL database
+* gcloud
+
+### Set the environment
+
+* Set some variables
+
+```bash
+export PROJECT_ID=(YOUR PROJECT ID)
+export PROJECT_NUMBER=(YOUR PROJECT NUMBER)
+export REGION=(REGION IN WHICH THINGS WILL BE DEPLOYED)
+export ZONE=(ZONE IN WHICH THINGS WILL BE DEPLOYED)
+export DB_INSTANCE_NAME=(NAME FOR POSTGRESQL INSTANCE)
+export DB_PASSWORD=(CRAZY COMPLICATED PASWORD FOR THE DB)
+export SERVICE_ACCOUNT_NAME="bigtable-autoscaler"
+export DB_CONNECTION_NAME=${PROJECT_ID}:${REGION}:${DB_INSTANCE_NAME}
+export BIGTABLE_INSTANCE_NAME=(Bigtable instance name)
+export BIGTABLE_CLUSTER_ID=(Bigtable cluster name)
+```
+
+### Set the Postgres DB
+
+* Set CloudSQL with Postgres
+
+```bash
+gcloud sql instances create ${DB_INSTANCE_NAME} \
+--database-version=POSTGRES_9_6 \
```

Comment:
Since we are doing this, maybe it's interesting to use Cloud SQL on Postgres version 13 instead of 9.6. 
What do you think?
It has a bunch of interesting performance improvements. 
E.g. Smaller indexes and quicker queries.

Reply:
I was not sure if there are any specific dependencies on version 9.6. If the dev team is OK with version 13, I can do the change :-)